Common/cpp/Fabric/PropsRegistry.h

```diff
@@ -26,19 +26,23 @@ class PropsRegistry {
   void remove(const Tag tag);
 
   void pleaseSkipCommit() {
```

Comment:
Should we specify who exactly should skip commit?
```suggestion
  void pleaseSkipReanimatedCommit() {
```